inc/autoload.function.php

```diff
@@ -380,7 +380,8 @@ function glpi_autoload($classname) {
    }
 }
 if ($needrun) {
-   die('Run "composer install --no-dev" in the glpi tree');
+   die('Run "composer install --no-dev" in the glpi tree.<br>'
+       . 'To install composer please refer to <a href="https://getcomposer.org/">https://getcomposer.org/</a>');
 }
```

Comment:
HTML is not acceptable here, as will look badly for CLI commands

Reply:
Ah yes, you're right, so I'll add a check with isCommandLine()